Add intro articles to Azure Kit (#1866)
vaadin/docs: articles/tools/azure/configuration.asciidoc

---
title: Configuration
description: Configuration options for Azure Kit.
order: 20
---

= Configuring Azure Kit

// Need a few configuration possibilities or scenarios,
// maybe in separate documents that link from this page.

// The text below comes from Tamas' README file on the demo.
// With some minor changes, it should be usable for this configuration document.

## Scale a Deployment

At http://localhost:8080/counter you find a counter which value is held in the UI.
Pushing the "Increment" button increments the counter and keeps a log of the operation, tracking the hostname and address of the node currently serving the request.

> Now let's simulate the unavailability of the pod handling the requests (assume it's the `azure-kit-demo-f87bfcbb4-5qjml` pod):

Reviewer: ```suggestion
You can simulate the unavailability of the pod handling the requests like this (assume it's the `azure-kit-demo-f87bfcbb4-5qjml` pod):
```

```
# kubectl delete pod azure-kit-demo-f87bfcbb4-5qjml
```

And wait until the pod is terminated and there's a new one running:

```
# kubectl get pods
NAME                                 READY   STATUS    RESTARTS      AGE
azure-kit-demo-f87bfcbb4-jj5c4       1/1     Running   0             12s
azure-kit-demo-f87bfcbb4-czkzr       1/1     Running   0             51s
azure-kit-demo-f87bfcbb4-gjqw6       1/1     Running   0             51s
azure-kit-demo-f87bfcbb4-rxvjb       1/1     Running   0             51s
azure-kit-redis-788d56c66-8b259      1/1     Running   0             51s
```

> If you try to increment the counter again, your request will be redirected to another pod but the session should be restored and the counter will keep incrementing.

Reviewer: ```suggestion
If you try to increment the counter again, your request is redirected to another pod but the session should be restored and the counter keeps incrementing.
```
